src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -278,10 +278,10 @@ private static boolean isDebugEnabled(BidRequest bidRequest, ExtBidRequest extBi
     }
 
     /**
-     * Checks if bidder name is valid in case when bidder can also be alias name.
+     * Checks if bidder name or alias name is valid and that bidder(and its alias) is active.
      */
     private boolean isValidBidder(String bidder, Map<String, String> aliases) {
-        return bidderCatalog.isValidName(bidder) || aliases.containsKey(bidder);
+        return bidderCatalog.isActive(resolveBidder(bidder, aliases));
```

Comment:
Pls add unit test for the case when disabled bidder comes in request to be ensure `org.prebid.server.spring.config.bidder.util.BidderDepsAssembler#ERROR_MESSAGE_TEMPLATE_FOR_DISABLED` will be present in response.

m.b. it make sense to check for NULL in place it is occurred.

Reply:
I've decided to do it another way - introduce the `Usersync.EMPTY` for disabled bidders to keep the flow before these changes and therefore keep `ERROR_MESSAGE_TEMPLATE_FOR_DISABLED` displayed later. Otherwise it would not occur as we simply filter disabled bidders out before getting to that point.